### ISSUE TITLE
Make background image responsive

### ISSUE THREAD

**Issue by ahmadabdolsaheb** [freeCodeCamp/coderadio-client#105]
The background and the Saron animation are fixed and do not change size based on viewport, so smaller screens see only a portion of the image and large fullhd + screen will see the image stock to a corner.

The optimal solution would be to show as much of the image as possible. For example:
If the screen is very tall, the height of the image should be set to the height of the screen and
if the screen is very wide, the width of the image should be set to the width of the screen.

It would be great if we could achieve this without using js.

**Comment by Ismailtlem** [comment]
I will work on this

**Comment by ahmadabdolsaheb** [comment]
Hi @Ismailtlem, i was wondering how your progress is going with the fix, and if you need help with it.

**Comment by Ismailtlem** [comment]
Hi,

Sorry for my late answer.
On the App.css code, I think that the 2 css rules responsible for the background image are those ones : 

.animation {
  background-image: url("https://cdn-media-1.freecodecamp.org/code-radio/bg.jpg");
  background-position: left bottom;
  background-repeat: no-repeat;
  position: relative;
  width: 100%;
  height: calc(100% - 158px);
}

.animation.saron::before {
  content: "";
  background-image: url("https://cdn-media-1.freecodecamp.org/code-radio/Saron3.gif");
  left: 388px;
  bottom: 0px;
  width: 509px;
  height: 767px;
  position: absolute;
}

What I tought of was adding some media queries after those 2 css rules. Something like for any mininmum height of 2000px (or some large screen size), set the height to 100vh. And for any minimum width of 2000px, set the width to 100vh. I am  not sure about which value set to the gif animation for those large screens.

**Comment by ahmadabdolsaheb** [comment]
feel free to change the elements as well. You might be able to create individual image elements and stack them on the top of one another so you could adjust their size independently.

**Comment by Ismailtlem** [comment]
Hi,

Maybe this rule can be removed : 

@media (max-width: 500px) {
  #visualizer {
    background-color: #002ead;
    top: 238px;
  }
}
It displays a blue background for width < 500 px

I think that adding background-size: 100% to the rule : 

#container {
  text-align: justify;
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: calc(50px + 38px);
}
would make the background image responsive.

I am looking for what to do regarding the animation.

**Comment by ahmadabdolsaheb** [comment]
So the reason that the background is blue is because we are not loading the background/animation for small screens to save data.

**Comment by Ismailtlem** [comment]
Hi,

Would removing the animation for small screens be an acceptable solution ?

**Comment by ahmadabdolsaheb** [comment]
Yes, we are already removing the animation and the background image for small screen since the animation is a large file.

**Comment by Ismailtlem** [comment]
Hi,

I tried to push some code but I get a permission error 
![image](https://user-images.githubusercontent.com/34961373/104511789-8d3b0e80-55ed-11eb-99c3-3e414df27736.png)

Are PR blocked ?

**Comment by Ismailtlem** [comment]
Any feedback about the solution proposed in the PR #121  ?
